src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt

```diff
@@ -0,0 +1,73 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.resources
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.welcometoprison.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.welcometoprison.service.PrisonService
+
+@RestController
+@Validated
+@PreAuthorize("hasRole('SYSTEM_USER')")
```

Comment:
I just stuck with SYSTEM_USER for now. Not sure if we can narrow this down? Tbh not entirely sure what "SYSTEM_USER" represents, assuming a DPS service?

Reply:
SYSTME_USER is a bit of a catch all role and probably does need narrowing down at some point.

Reply:
Maybe worth discussing tomorrow but I think we could possibly use the role we already have? 

`ROLE_VIEW_INCOMING_MOVEMENTS`